asf_tools/README.md

```diff
@@ -28,7 +28,23 @@ python -m pip install asf_tools
 
 ## Quick Usage
 
-TBD
+### Local Resolution Weighted Composite
+
+The `make_composite` tool allows you to create a local-resolution-weighted composite from a set of Sentinel-1 RTC
+products ([D. Small, 2012](https://doi.org/10.1109/IGARSS.2012.6350465)). It is intended to be used with RTC products generated by [ASF HyP3](https://hyp3-docs.asf.alaska.edu/using/vertex)
+
+You will need to request RTC products using the `Include Scattering Area` option, then download and unzip them into an empty directory.
+
+
+To generate a composite of the co-polarization image navigate to the directory containing the unzipped RTC products and run this command.
```

Comment:
"co-polarization image navigate to"
* images plural
* comma after image

"run this command." --> "run:"